docs/architecture/decisions/0006-use-css-custom-properties-for-component-styles.md

```diff
@@ -0,0 +1,32 @@
+# 6. Use CSS custom properties for component styles
+
+Date: 2020-04
+
+## Status
+
+Accepted
+
+Enhanced by [8. Evolve tokens in backward compatible manner](0008-evolve-tokens-in-backward-compatible-manner.md)
+
+Enhanced by [8. Evolve tokens in backward compatible manner](0008-evolve-tokens-in-backward-compatible-manner.md)
+
+Enhanced by [8. Evolve tokens in backward compatible manner](0008-evolve-tokens-in-backward-compatible-manner.md)
```

Comment:
These all appear to the be the same?

Reply:
Ah, this was from a test I performed. I'll clean it up!